pipeline/pipeline-jobs.yml

```diff
@@ -0,0 +1,46 @@
+resources:
+- name: census-worth-self-help
+  type: git
+  source:
+    uri: https://github.com/ONSdigital/census-worth-self-help.git
+    branch: feature/ONS-152-pipeline-cleanup
+  webhook_token: ((git.census-worth-self-help-webhook-token))
+  check_every: 15m
+
+- name: content-repo
+  type: git
+  source:
+    uri: git@github.com:ONSdigital/census-worth-self-help-test.git
+    branch: master
+    private_key: ((git-keys-test.private-key))
+  webhook_token: ((git.census-worth-self-help-test-webhook-token))
+
+jobs:
+- name: test
+  plan:
+  - get: census-worth-self-help
+    trigger: true
+    
+  - task: test
+    params:
+      SERVICE: default
+      BACKEND_BRANCH: master
```

Comment:
By BACKEND_BRANCH I believe you're meaning the branch that Netlify CMS will merge into.  This will always be master now (the test_content branch is on it's way out).  I'd just bake master in to the yml script.

Reply:
done